gallery_widgets/lib/features/wrap_chip/widgets/main_view.dart

```diff
@@ -0,0 +1,167 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:gallery_widgets/features/wrap_chip/cubit/wrap_chip_cubit.dart';
+
+class MainView extends StatelessWidget {
+  const MainView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 20, horizontal: 10),
+      child: BlocBuilder<WrapChipCubit, WrapChipState>(
+        builder: (context, state) {
+          return ListView(
+            children: [
+              buildChipPart(
+                chipType: 1,
+                state: state,
+                title: 'Chip Chip',
+                chipTexts: ['Chip', 'ActionChip', 'RawChip'],
+                selectedIndex: state.selectedChip,
+                onSelected: (index) {
+                  context.read<WrapChipCubit>().onChangeSelectedChip(
+                        type: 1,
+                        value: index,
+                      );
+                },
+              ),
+              const SizedBox(height: 15),
+              buildChipPart(
+                chipType: 2,
+                state: state,
+                title: 'Choice Chip',
+                chipTexts: ['Disable', 'Small', 'Large'],
+                selectedIndex: state.selectedChoiceChip,
+                onSelected: (index) {
+                  context.read<WrapChipCubit>().onChangeSelectedChip(
+                        type: 2,
+                        value: index,
+                      );
+                },
+              ),
+              const SizedBox(height: 15),
+              buildChipPart(
+                chipType: 3,
+                state: state,
+                title: 'Input Chip',
+                chipTexts: ['Disable', 'ios', 'Android'],
+                selectedIndex: state.selectedInputChip,
+                onSelected: (index) {
+                  context.read<WrapChipCubit>().onChangeSelectedChip(
+                        type: 3,
+                        value: index,
+                      );
+                },
+              ),
+            ],
+          );
+        },
+      ),
+    );
+  }
+
+  Widget buildChipPart({
+    required int chipType,
+    required String title,
+    required List<String> chipTexts,
+    required int selectedIndex,
+    required Function(int) onSelected,
+    required WrapChipState state,
```

Comment:
You should not use the `state` like this => use BlocBuilder for every widget using the `state`